app/models/assessment.rb

```diff
@@ -1,14 +1,31 @@
 # coding: utf-8
 class Assessment < ApplicationRecord
 
-  belongs_to :format
-  belongs_to :created_by, class_name: 'Member'
+  belongs_to :member
+  has_many :questions, inverse_of: :assessment
+  has_many :other_versions,
+           -> (a) { where.not(id: a.id) },
```

Comment:
Won't this return all other assessments, not just different versions?  Please add a spec testing that this gives just the assessments you expect.

Reply:
I don't think it will, because the foreign_key and primary_key must also match, this just adds an additional condition. I'll add spec tho